frontend/server/src/test/resources/snapshots/snapshot1.cfg

```diff
@@ -9,7 +9,7 @@ default_workers_per_model=4
 model_store=../archive/src/test/resources/models
 async_logging=true
 number_of_gpu=0
-model_snapshot={\n  "name"\: "20200329045334828-startup.cfg",\n  "modelCount"\: 1,\n  "created"\: 1585457614832,\n  "models"\: {\n    "noop"\: {\n      "1.11"\: {\n        "defaultVersion"\: true,\n        "marName"\: "noop.mar",\n        "minWorkers"\: 4,\n        "maxWorkers"\: 4,\n        "batchSize"\: 1,\n        "maxBatchDelay"\: 100,\n        "responseTimeout"\: 120\n      }\n    }\n  }\n}
+model_snapshot={\n  "name"\: "20200329045334828-startup.cfg",\n  "modelCount"\: 1,\n  "created"\: 1585457614832,\n  "models"\: {\n    "noop"\: {\n      "1.11"\: {\n        "defaultVersion"\: true,\n        "marName"\: "noop.mar",\n        "minWorkers"\: 4,\n        "maxWorkers"\: 4,\n        "batchSize"\: 1,\n        "maxBatchDelay"\: 100,\n        "responseTimeout"\: 120,\n        "runtimeType"\: "python"\n     }\n    }\n  }\n}
```

Comment:
What happens if we do not have RuntimeType="python" in the cfg file? Is this covered with a test as well?